Xamarin.Forms.Platform.UAP/MasterDetailControl.cs

```diff
@@ -100,7 +100,7 @@ public Windows.Foundation.Size DetailSize
 						width -= _masterPresenter.ActualWidth;
 				}
 
-				return new Windows.Foundation.Size(width >= 0 ? width : 0, height);
+				return new Windows.Foundation.Size(width > 0 ? width : 0, height);
```

Comment:
This could still fail if `ActualHeight` is between zero and `_commandBar.ActualHeight`; height could end up being less than zero. 

What if we reverted line 92 and in 103 did 
`return new Windows.Foundation.Size(width > 0 ? width : 0, height > 0 ? height : 0)`
instead? (Or used `Math.Max(height, 0)`.)